### ISSUE TITLE
Deux héritages de même type dans un MCD

### ISSUE THREAD

**Issue by fduchatea** [laowantong/mocodo#64]
Mocodo version 3.0.6 : l'héritage rend vraiment Mocodo complet.

Dans un MCD, si on veut mettre 2 héritages de même type (XT par ex), Mocodo génère l'erreur suivante : 

> Err.7 - Association « XT » dupliquée. Si vous souhaitez faire apparaître deux associations sous le même nom, vous devez suffixer celui-ci avec un nombre.

Si on utilise XT et XT2 comme suggéré : 

> Err.24` - Spécialisation inconnue « XT2 ».

Si on remplace l'un des héritages XT par X ou T ou rien, alors ça fonctionne. Le fait que les spécialisations soient assimilées à des associations (en interne) semble contraignant. Peut-être autoriser de suffixer les spécialisations comme pour les associations afin de résoudre le problème ?

Exemple de code Mocodo (minimal) avec deux XT :

```
Utilisatrices : idU, email
Invitées : adresseIP
UtilisatricesPlus : nom, prénom,
Inscrites :
Abonnées : 
/XT\ Utilisatrices ->> Invitées, UtilisatricesPlus : catégorie
/XT\ UtilisatricesPlus ->> Inscrites, Abonnées : catégorie
```

**Comment by laowantong** [comment]
Oui, effectivement, suffixer avant ou après l'antislash ne marche pas comme attendu. Je n'avais pas pensé à ce cas. Je vais me pencher là-dessus et ajouter dans mes tests votre exemple, plongé comme ci-dessous:

```
:
:
Inscrites :
:

:
UtilisatricesPlus : nom, prénom
/XT\ UtilisatricesPlus ->> Inscrites, Abonnées : catégorie
Abonnées :

Utilisatrices : idU, email
/XT\ Utilisatrices ->> Invitées, UtilisatricesPlus : catégorie
Invitées : adresseIP
:
```

**Comment by laowantong** [comment]
Résultat :

![image](https://user-images.githubusercontent.com/3048819/202662863-6bf5d58a-86c1-442c-bf38-9ada88051e50.png)
